javascript/javascript1/week1/housey-pricey.js

```diff
@@ -0,0 +1,44 @@
+// Variables we defined for the task
+const pHouseDepth = 10;
+const pHouseHeight = 10;
+const pHouseWidth = 8;
+const pGardenSize = 100;
+
+const jHouseDepth = 11;
+const jHouseWidth = 5;
+const jHouseHeight = 8;
+const jGardenSize = 70;
+
+// Multipliers given by task for calculating the price
+const houseSizeMultiplier1 = 2.5;
+const houseSizeMultiplier2 = 1000;
+const gardenSizeMultiplier = 300;
```

Comment:
Great! It may seem excessive now but as functions and codebases grow it makes it much easier to refer to in the future and for collaborators!